cabal-install/Distribution/Client/CmdConfigure.hs

```diff
@@ -58,10 +58,10 @@ configureAction (configFlags, configExFlags, installFlags, haddockFlags)
         ( globalFlags, configFlags, configExFlags
         , installFlags, haddockFlags )
         PreBuildHooks {
-          hookPrePlanning = \projectRootDir _ cliConfig ->
+          hookPrePlanning = \rootDir _ cliConfig ->
```

Comment:
Yes, just avoiding name clashes. Probably better, as I mentioned, to put this into the DistDirLayout